main/src/main/java/cgeo/geocaching/log/LogEntry.java

```diff
@@ -57,7 +57,7 @@ public class LogEntry implements Parcelable {
     /** The log message */
     @NonNull public final String log;
     /** The log date */
-    public final long date;
+    public long date;
```

Comment:
I made this member editable, because the objects are observable and creating a new one instead of changing an existing one wouldn't work

Reply:
This breaks `LogEntry` from being immutable although it still "looks like" it is. 

Setting date should normally work with

`logEntry.buildUpon().setDate().build()`

(This creates a new `LogEntry` instance of course)

That doesn't work in your case?

(I am not against making Log entry mutable, but then it should be done for whole class which is a bigger task)